examples/computer_vision/detectron2_coco_pytorch/const.yaml

```diff
@@ -0,0 +1,23 @@
+name: detectron2_const
+environment:
+    image: "katport/detectron2:cuda-10.1"
```

Comment:
As of now, I'm gonna use my Determined account since thats what we do in examples currently. I have it on our AMLE agenda to discuss migrating to maybe a specific docker hub.